pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md

```diff
@@ -0,0 +1,317 @@
+---
+pagename: Conversation Tester
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+permalink: third-party-bots-conversation-tester.html
+indicator: both
+---
+
+#### Conversation tester.
+## Introduction
+By using the Conversation tester, you will be able to test your bot and identified if your bot's response have the right configuration. 
+The Conversation Tester feature aims to help you to identified possible missconfiguration in your bot. This provides three important components. First the chat window in which you can start testing your bot. Second the Logs column in which you will see if you bot response was process or had an error. And finally the JSON EDITOR column that will help you to identified the error cause.
+
+## Processed Response
+When you are testing you bot and you get a successful response, it means that your bot's response has the correct configuration. See the image below.
+<img style="width:800px" src="img/botconnectordashboard/processed_response.png">
+
+## Error Response
+When testing your bot and you get an error in the response, you will be able to see what was the JSON object that produced the error in your bot response. By hovering on the last alert you will see a tooltip that will give you the possible solutions for your error. See the image below.
+
+
+<img style="width:800px" src="img/botconnectordashboard/error_response.png">
+
+### Errors that can be found per vendor
+
+### Dialog Flow V2
+
+## Dialog Flow V2 Invalid Action Body
```

Comment:
Are you going to show the exact same text in conversation tester UI if this error happened? Just wanted to make sure that users are able to make text connection to what they see in the conversation tester UI

Reply:
It is not exactly the same text. For example when I have an Action body error, the error will say 
Unable to parse action due to the invalid body

Reply:
Ok so you need to update it in UI as well in such a way that user know and able to make the link

Reply:
@fahimmahmoodmir  In the UI I have the error message, for example "Unable to parse action due to the invalid body", that is the description the user needs in the UI, here in the documentation I have the error type as title and the explanation. I don't think I have to change the error message in the UI.

Reply:
Ok if you think the user will be able to make the link between UI message and the documentation then its fine